### PR TITLE
Get board

### PR DESCRIPTION
**Pull request by thoreyjona**
# Pull request
## :heavy_exclamation_mark: Check list
- [X] Did you follow the TDD protocol?  
- [X] Did your code build?  
- [X] Did your code pass through Travis?  
- [X] Did you write a description about your branch and its purpose?
## :heavy_exclamation_mark: Description and purpose of the branch:

Implemented and tested constructors and a few functions

Signature
💯

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/PowerRangersHug/TicTacToe/pull/12" target="_blank">ReviewNinja

File: src/main/java/is/TicTacToe/data/Game.java [hunk 1]
```diff
@@ -4,12 +4,87 @@
...
+    public void MakeMove(int x, int y, String playerName)
+    {
+        Player player = GetPlayer(playerName);
+        if (player == null)
+        {
+            throw new IllegalArgumentException("Invalid Player Name!");
+        }
+
+        String symbol = player.GetSymbol();
+        System.out.println(symbol);
```
Comment (fanneykristjans) [inline src/main/java/is/TicTacToe/data/Game.java]: maybe remove this debugging print?

Comment (thoreyjona) [reply src/main/java/is/TicTacToe/data/Game.java]: On it! 👍

**Review by fanneykristjans — APPROVED** [review]
Very good, we will just implement the GetWinner function in a branch specific for finishing a game / checking who won.

**Pull request merged**